planet/cli/subscriptions.py

```diff
@@ -113,13 +113,13 @@ async def update_subscription_cmd(ctx, subscription_id, request, pretty):
         echo_json(sub, pretty)
 
 
-@subscriptions.command(name='describe')
+@subscriptions.command(name='get')
 @click.argument('subscription_id')
 @pretty
 @click.pass_context
 @translate_exceptions
 @coro
-async def describe_subscription_cmd(ctx, subscription_id, pretty):
+async def get_subscription_cmd(ctx, subscription_id, pretty):
     """Gets the description of a subscription and prints the API response."""
```

Comment:
```suggestion
    """Get the description of a subscription."""
```
This is short enough to fit entirely in `planet subscriptions --help`.